rdbms/src/main/java/org/spine3/server/storage/jdbc/command/query/UpdateCommandQuery.java

```diff
@@ -55,9 +55,9 @@ private UpdateCommandQuery(Builder builder) {
     public static Builder newBuilder() {
         final Builder builder = new Builder();
         builder.setStatusIndexInQuery(2)
-                .setIdIndexInQuery(3)
-                .setRecordIndexInQuery(1)
-                .setQuery(QUERY_TEMPLATE);
+               .setIdIndexInQuery(3)
```

Comment:
Please have something (like an `enum`?) to store the indexes instead of using the integer values straightaway in the code.